addons/crm/static/src/js/tours/crm.js

```diff
@@ -1,100 +1,113 @@
-    position: 'right',
-    edition: 'community'
+    content: _t('Ready to boost your sales? Let\'s have a look at your <b>Pipeline</b>.'),
+    position: 'bottom',
+    edition: 'community',
 }, {
     trigger: '.o_app[data-menu-xmlid="crm.crm_menu_root"]',
-    content: _t("Ready to boost your sales? Your <b>Pipeline</b> can be found here, under the <b>CRM</b> app."),
+    content: _t('Ready to boost your sales? Let\'s have a look at your <b>Pipeline</b>.'),
     position: 'bottom',
     edition: 'enterprise',
 }, {
-    trigger: ".o-kanban-button-new",
+    trigger: '.o-kanban-button-new',
     extra_trigger: '.o_opportunity_kanban',
-    content: _t("Click here to <b>create your first opportunity</b> and add it to your pipeline."),
-    position: "bottom",
+    content: _t("<b>Create your first opportunity.</b>"),
+    position: 'bottom',
 }, {
-    trigger: ".o_kanban_quick_create input:first",
-    content: _t("<b>Choose a name</b> for your opportunity, example: <i>'Need a new website'</i>"),
+    trigger: ".o_kanban_quick_create .o_field_widget[name='partner_id']",
+    content: _t('Look for a Contact or create a new one.<br><i>Tip : Did you know you can search by VAT Number as well?</i>'),
     position: "right",
+    run: function (actions) {
+        actions.text("Brandon Freeman", this.$anchor.find("input"));
+    },
+}, {
+    trigger: ".o_kanban_quick_create .o_field_widget[name='name']",
+    auto: true,
+}, {
+    trigger: '.o_kanban_quick_create .o_field_monetary[name="planned_revenue"]',
+    content: _t("Define here the Expected Revenue of this Opportunity."),
+    position: 'right',
 }, {
     trigger: ".o_kanban_quick_create .o_kanban_add",
-    content: _t("Click here to <b>add your opportunity</b>."),
+    content: _t("Now, <b>add your Opportunity</b> to your Pipeline."),
     position: "bottom",
 }, {
-    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child",
+    trigger: ".o_opportunity_kanban .o_kanban_group:first-child .o_kanban_record:last-child .oe_kanban_content",
+    extra_trigger: ".o_opportunity_kanban",
     content: _t("<b>Drag &amp; drop opportunities</b> between columns as you progress in your sales cycle."),
-    position: "right",
+    position: "bottom",
     run: "drag_and_drop .o_opportunity_kanban .o_kanban_group:eq(2) ",
 }, {
     trigger: ".o_kanban_record:not(.o_updating) .o_activity_color_default",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("This opportunity has <b>no activity planned</b>."),
-    position: "bottom"
+    content: _t("Looks like nothing is planned. :(<br><br><i>Tip : Schedule activities to keep track of everything you have to do!</i>"),
+    position: "bottom",
 }, {
     trigger: ".o_schedule_activity",
     extra_trigger: ".o_opportunity_kanban",
-    content: _t("Let's schedule an activity."),
-    position: "bottom"
+    content: _t("Let's <b>Schedule an Activity.</b>"),
+    position: "bottom",
+    width: 200,
```

Comment:
Is that an actual tip property? Never seen it but it's possible.

Reply:
yes it is. The PO asked for a smaller box for this tip

FYI : width – width in px of the tip when opened, default 270